src/core/app-template.cc

```diff
@@ -117,6 +117,17 @@ const app_template::seastar_options& app_template::options() const {
 
 app_template::configuration_reader app_template::get_default_configuration_reader() {
     return [this] (bpo::variables_map& configuration) {
+        auto seastar_conf = std::getenv("SEASTAR_CONF");
```

Comment:
how about changing scylla instead:

```c++
app.set_configuration_reader([cfg, &app](bpo::variables_map& configuration) {
    auto seastar_conf = cfg->get_conf_sub("seastar.conf");
    if (std::ifstream ifs(seastar_conf); ifs) {
        bpo::store(bpo::parse_config_file(ifs, app.get_conf_file_options_description()), configuration);
    }
});
```

in `scylla_main()`. probably we need use the facilities and mechinaries provided by seastar, instead of arming it with them to implement the policies.